src/Microsoft.Graph.Core/CoreConstants.cs

```diff
@@ -138,7 +141,28 @@ public enum HttpMethods
             /// <summary>
             /// The DELETE method.
             /// </summary>
-            DELETE
+            DELETE,
```

Comment:
nit: move the enum to it's own file (not under the core constants class) to simplify imports

Reply:
Thanks. This is now resolved via https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/pull/218/commits/6010bc263ba39bd484e091c5ece90f0fc6dfc6e4